.travis.yml

```diff
@@ -72,9 +72,30 @@ matrix:
     - name: "Linux with Defaults against GCC Network and Service Tests"
       os: linux
       compiler: gcc
-      env: BUILD_TARGET="linux-auto-gcc-check-happy"
+      env: BUILD_TARGET="linux-auto-gcc-check-happy" TRAVIS_TAG="true"
```

Comment:
What does "TRAVIS_TAG=true" do?

Reply:
I added information for deploy part, below line is there: 
tags: true: deployment is triggered if and only if $TRAVIS_TAG is set here.
Any suggestion to make it more clear?